packages/dev/core/src/Materials/imageProcessing.ts

```diff
@@ -0,0 +1,197 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { serializeAsImageProcessingConfiguration } from "../Misc/decorators";
+import type { Nullable } from "../types";
+import type { ImageProcessingConfiguration } from "./imageProcessingConfiguration";
+import type { Observer } from "../Misc/observable";
+import type { BaseTexture } from "./Textures/baseTexture";
+import type { ColorCurves } from "./colorCurves";
+
+// Explicit re-export of types to help TypeScript resolve them in declaration files
+export type { Observer } from "../Misc/observable";

```

Comment:
I was getting errors in the `npm run build:es6` involving `../..` import paths in this mixin.
An AI tool came up with the solution of adding these explicit exports. I don't really understand it but it works. It should be checked by someone who understands issues like this.